src/main/java/step4/domain/Data.java

```diff
@@ -0,0 +1,29 @@
+package step4.domain;
+
+public class Data {
+    final static int defaultCount = 5;
+    final String names;
+    final int count;
```

Comment:
```suggestion
    private final static int DEFAULT_COUNT = 5;
    private final String names;
    private final int count;
```

게터 메서드를 선언했다면, 내부 맴버는 private 으로 해주세요.
또한 final static 상수는 대문자 snake 방식으로 이름짓는게 관례입니다.